### DISCUSSION TITLE
Is there a way to build without assistant suppor?

### DISCUSSION THREAD

**Discussion by kinnison** [zed-industries/zed#14739] [Feature Requests]
Hi,

I work somewhere that LLM use are banned for various reasons which I am not in control of.  While it's easy to say "Do not use the assistant, it'd be far safer if we could offer a build of Zed internally which does not even have that feature so that it can't be accidentally turned on.  Is there a cargo feature I can disable which removes this functionality?

D.

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support

**Comment by kinnison** [comment]
Oh well, never mind.
